tests/test_fields.py

```
@@ -0,0 +1,138 @@
+
+def test_field_verbose_name():
+    user = Person("john_doe", 30)
+    serialized = dumps(user)
+    print(serialized)
+    data = loads(serialized)
+    assert data.original.username == "john_doe"
+    assert data.original.age == 30
+    assert data.original.__class__.__name__ == "Person"
+
+
+def test_nofield_typechecking():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_nofield_typechecking`

Add return type annotation: `None`

(ANN201)